test/e2e_node/resource_usage_test.go

```diff
@@ -142,15 +147,78 @@ func runResourceUsageTest(f *framework.Framework, rc *ResourceCollector, testArg
 		// sleep for an interval here to measure steady data
 		sleepAfterCreatePods = 10 * time.Second
 	)
-	pods := newTestPods(testArg.podsNr, true, framework.GetPauseImageNameForHostArch(), "test_pod")
+	framework.Logf("creating namespaces")
+	var namespaces []*v1.Namespace
+	for i := 0; i < testArg.podsNr; i++ {
```

Comment:
Yes, multiple namespaces are not needed as long as we have explicit secrets set in the pod. In fact, multiple namespaces are usually not hard to clean up in an e2e test.